incubator/hnc/internal/object/object_test.go

```diff
@@ -1,9 +1,10 @@
 package object
 
 import (
-	. "github.com/onsi/gomega"
 	"testing"
 
+	. "github.com/onsi/gomega"
+
```

Comment:
Ah right, it should be in the 2nd section. Could you please remove the empty line though?

Reply:
done